syncthing/build-syncthing.py

```diff
@@ -38,7 +37,6 @@
         'goarch': 'amd64',
         'jni_dir': 'x86_64',
```

Comment:
How is this related to SDK31 - just that the workaround of copying some build artifacts around doesn't work anymore?  
Generally my latest comment there still stands - I don't really want to spend time on android, as in if this is only an issue for chromebooks I unfortunately don't care. The most likely explanation is anyway that something in the NDK/SDK stack just isn't how go expects it to be, thus it's breaking - not much we can do there.

Reply:
Since bumping api level to 31, I am unsuccessful at running the app on the emulator. (Even though I'm trying this with the fork but it should be the same for the official app as archs are the same.)

Reply:
It's okay , thanks for looking into it. Christmas first ;-). My time is limited too, so we can do it somewhen later. I put on my list for next year :-).